web/app/js/components/Tap.jsx

```diff
@@ -90,7 +90,7 @@ class Tap extends React.Component {
       ...query
     }));
     this.setState({
-      webSocketRequestSent: true,
+      tapRequestInProgress: true,
```

Comment:
It's no problem to set this into the state, but for my own understanding, isn't this redundant now that you're setting `tapRequestInProgress` when `startTapSteaming` is called, before the websocket is opened?

Reply:
yeah totally!